nikola/plugins/task/authors.py

```diff
@@ -138,6 +140,10 @@ def provide_context_and_uptodate(self, author, lang, node=None):
         kw.update(context)
         return context, kw
 
+    def get_other_language_variants(self, author, lang, classifications_per_language):
```

Comment:
(Codacy’s *Parameters differ from overridden 'get_other_language_variants' method* suggestion here and in other places is pretty sound)